Updated Order and Refund Events

clintonb opened the pull request.

clintonb commented:
We should use Segment's semantic events to avoid future confusion:

* https://segment.com/docs/spec/ecommerce/v2/
* https://segment.com/docs/integrations/google-analytics/

clintonb commented:
@edx/ecommerce @edx/helio please review

clintonb commented:
Google Tag Manager has been updated to recognize the "Order Completed" event in addition to "Completed Order" and "Completed Purchase".

mjfrey commented on ecommerce/static/js/pages/receipt_page.js:
```diff
@@ -9,7 +9,7 @@ define([
 
         function trackPurchase(order_id, total_amount, currency) {
-            window.analytics.track('Completed Order', {
+            window.analytics.track('Order Completed', {
```
Do we need to tell Adley about this change?

clintonb replied:
I have no idea.

mjfrey replied:
I would just let BA know about the change in case it affects the reporting.

clintonb replied:
Agreed. I have notified them via HipChat.

douglashall commented:
@clintonb What needs to be updated in GA/GTM in order to support this change? There are WL containers and properties in play that we need to consider.

clintonb commented:
@douglashall I made the following changes in GTM:

1. Renamed the "Completed Purchase" trigger to "Order Completed" (not required, but good for consistency).
2. Added a regex for the event name: `Completed (Order|Purchase)|Order Completed`
